org.civicrm.bootstrapcivicrm/gulpfile.js

```diff
@@ -1,20 +1,41 @@
+require('es6-promise').polyfill();
 var gulp = require('gulp');
 var bulk = require('gulp-sass-bulk-import');
 var rename = require('gulp-rename');
 var sass = require('gulp-sass');
+var postcss = require('gulp-postcss');
+// Used to add necessary prefixes at the css level
+var postcssPrefix = require('postcss-prefix-selector');
+// Used to remove duplicated css rules
+var postcssDiscardDuplicates = require('postcss-discard-duplicates');
 
 gulp.task('sass', function () {
```

Comment:
Given the folder structure change i mentioned in a comment, i would rename this task into `sass-bootstrap`